Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/AI/NavMeshQueryFilterTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using UnityEngine.AI;
+
+namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes.AI
+{
+    public class NavMeshQueryFilterTests : ValueTypeTester<NavMeshQueryFilter>
+    {
+        private static readonly PropertyInfo _costsProperty = typeof(NavMeshQueryFilter).GetProperty("costs", BindingFlags.NonPublic | BindingFlags.Instance);
+
+        public static readonly IReadOnlyCollection<(NavMeshQueryFilter deserialized, object anonymous)> representations = new (NavMeshQueryFilter, object)[] {
+            (new NavMeshQueryFilter(), new {
+                costs = new float[] {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5420094)